android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -1023,4 +1023,27 @@ public AccessibilityManager getAccessibilityManager()
 	public void verifyCustomModules(TiRootActivity rootActivity)
 	{
 	}
+
+	public void popToRootWindow()
+	{
+		/**
+		 * emulates the iOS Tab.popToRootWindow() by closing all windows above a TabGroup.
+		 */
+		boolean isTabGroup = (activityStack.get(1).get() instanceof TiActivity)
+			&& ((TiActivity) activityStack.get(1).get())
```

Comment:
Hardcoding the index could potentially cause issues

Reply:
You are right. Especially if once we merge https://github.com/tidev/titanium-sdk/pull/13889 it could be 0 instead of 1. I'll try to loop and check for the ID